crux/src/Crux/Goal.hs

```diff
@@ -116,16 +116,18 @@ proveGoals opts _ctxt Nothing =
 proveGoals opts ctxt (Just gs0) =
   do let sym = ctxt ^. ctxSymInterface
      sp <- getSolverProcess sym
-     goalNum <- newIORef (0,0) -- total, proved
+     goalNum <- newIORef (0,0,0) -- total, proved, disproved
      nameMap <- newIORef Map.empty
      unless hasUnsatCores $
       sayWarn "Crux" "Warning: skipping unsat cores because MC-SAT is enabled."
      res <- inNewFrame sp (go sp goalNum gs0 nameMap)
-     (tot,proved) <- readIORef goalNum
+     (tot,proved,disproved) <- readIORef goalNum
```

Comment:
Should `tot = proved + disproved`? If so, it might be nice to add an assertion to that effect, just in case we forget to add to the counters somewhere, etc.